src/content/app/tools/blast/state/blast-api/blastApiSlice.ts

```diff
@@ -62,9 +64,15 @@ const blastApiSlice = restApiSlice.injectEndpoints({
       BlastSubmissionPayload
     >({
       query(payload) {
+        // merge the sequence and header using tofasta
+        const submittedSequences = payload.sequences.map((item) => ({
```

Comment:
Technically, we haven't yet submitted these sequences. We are about to. Maybe `querySequences` instead of `submittedSequences`?